.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 python:
-  - "3.5"
+  - "3.8"
 env:
   matrix:
   - TOXENV=py27
```

Comment:
We can also drop python2 compatibility

Reply:
I think that should be done in a follow up, to keep this about formatting